docs/release.md

```diff
@@ -39,28 +39,45 @@ These artifacts are provided in their respective channels and their natural way
 of distribution.
 
 # Cadence and Timeline
-Both a new release branch and initial release candidate are cut on the **first
-business day of each month.**
-
-If no blocker issues are discovered the release candidate is **promoted to a full
-release after 5 business days.**
+New release branches are cut 3 times a year, so a release cycle will occupy about 15 weeks.
+This release cycle mirrors the Kubernetes' and allows KubeVirt to release after the community.
+This looks like:
+| Week Number in Year | K8s Release Number | K8s Release Week | KubeVirt Release Week |                     Note                    |
+|:-------------------:|:------------------:|:----------------:|-----------------------|:-------------------------------------------:|
+| 1                   | 1.24               | 1 (January 03)   | 1 (~January 03)       |                                             |
+| 15                  | 1.24               | 15 (April 12)    | 15 (~April 12)        |                                             |
+| 17                  | 1.25               | 1 (April 26)     | 1 (~April 26)         | KubeCon + CloudNativeCon EU likely to occur |
+| 32                  | 1.25               | 15 (August 09)   | 15 (~August 09)       |                                             |
```

Comment:
The roughly-at-the-same-day like k8s date is a little bit unspecific to me. Can we make this more clear? Like 2 weeks after k8s, 4 weeks after k8s, or something like this?

It is sometimes tough to provide based on release candidates from k8s, I would want to on the one hand don't stress people too much, but on the other hand have a clear date on when we expect to have everything tested and ready.